internal/compiler/passes/resolving.rs

```diff
@@ -76,6 +76,24 @@ fn resolve_expression(
     }
 }
 
+/// Call the visitor for each children of the element recursively, starting with the element itself
+///
+/// The item that is being visited will be pushed to the scope and popped once visitation is over.
+fn recurse_elem_with_scope(
+    elem: &ElementRc,
+    scope: ComponentScope,
+    vis: &mut impl FnMut(&ElementRc, &ComponentScope),
+) -> ComponentScope {
+    let mut scope = scope;
```

Comment:
(you can just use `mut scope` in the function declaration)

Reply:
thanks, fixed

Reply:
now actually fixed, TIL about this syntax - unexpected :)